corehq/apps/users/role_utils.py

```diff
@@ -33,22 +18,23 @@ def archive_custom_roles_for_domain(domain):
 
 
 def unarchive_roles_for_domain(domain):
-    all_roles = UserRole.by_domain(domain, include_archived=True)
+    all_roles = SQLUserRole.objects.filter(domain=domain, is_archived=True)
```

Comment:
nit: this isn't really all roles anymore, just archived roles.

Reply:
```suggestion
    archived_roles = SQLUserRole.objects.filter(domain=domain, is_archived=True)
```